poshc2/client/command_handlers/ImplantHandler.py

```diff
@@ -477,6 +491,49 @@ def do_show_urls(user, command):
     clear()
 
 
+def do_show_hosted_files(user, command):
+    files = update_cache_urls()
+    filesformatted = "ID  URI  FilePath  ContentType  Base64  Active\n"
+    for i in files:
+        filesformatted += "%s  %s  %s  %s  %s %s \n" % (i[0], i[1], i[2], i[3], i[4], i[5])
+    print_good(filesformatted)
+    input("Press Enter to continue...")
+    clear()
+
+
+def do_add_hosted_file(user, command):
+    URI = input("URI Path: downloads/2020/application.docx ")
+    FilePath = input("File Path: .e.g. /tmp/application.docx ")
+    ContentType = input("Content Type: .e.g. text/html ")
+    Base64 = input("Base64 Encode File (Yes/No): ")
```

Comment:
Like this option but would make it (y/N) format to match elsewhere and make it clear what the default is -> we should also check the value before inserting it in the DB. Can use similar code to that in `startdaisy`.